docs/pages/guides/understanding-relationships.mdx

```diff
@@ -0,0 +1,323 @@
+import { Markdown } from '../../components/Page';
+
+# Understanding Relationships
+
+Relationships are the connections you make between different lists of content in Keystone. What you build depends a great deal on what you need. This guide will show you how to reason about, and configure relationships in Keystone so you can bring value to your project through structured content.
+
+---
+
+## How to reason about relationships
+
+Keystone provides a lot of flexibility when it comes to relationships. To get what you need there are two key questions you need to answer:
+
+1. **Which side of the relationship do I need to access data from?**
+
+Your needs here will define whether your relationship needs to be [one, or two-sided](#one-sided-and-two-sided-relationships).
+
+2. **How many connections do need on either side of my relationship?**
+
+Understanding this will determine the kind of [cardinality](#establishing-cardinality) you need to configure. 
+
+These topics are easier to understand by example. We’ll explore them, as well as the basics of setting up relationships through the use case of a **blog app** that has a `post` type for blog entries, and `user` type for post authors. Let’s get started…
+
+## How to define a relationship in Keystone
+
+Relationships are made using the [`relationship`](../apis/fields#relationship) field type within a [`list()`](../apis/schema). In our blog example we can connect a blog `post` to some `users` using the relationship field’s `ref` configuration option like so:
+
+
+```typescript
+import { config, createSchema, list } from '@keystone-next/keystone/schema';
+import { text, relationship } from '@keystone-next/fields';
+
+export default config({
+  lists: createSchema({
+    User: list({ fields: { name: text() } }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        authors: relationship({
+          ref: 'User',
+          // ↑ references the 'User' list from the 'author' field in the 'Post' list
+          many: true
+          // ↑ sets cardinality, see 'Establishing Cardinality' in this guide
```

Comment:
Comments should live above the code they reference.

I'm not actually sure we want to have comments here. I'd prefer to leave the explanation of what theses options do to the prose, rather than busying up the example 🤔 

```suggestion
          // references the 'User' list from the 'authors' field in the 'Post' list
          ref: 'User',
          // sets cardinality, see 'Establishing Cardinality' in this guide
          many: true,
```

Reply:
thanks for the heads-up re comments location 👍 

i'll commit this and return to the comments Vs prose thing in a separate commit.

think in this case the presence of the `many: true` option forced us into talking about cardinality before it was ideal. 

might use a callout to speak to that outside of the main narration